R/derive_merged.R

```diff
@@ -347,18 +385,26 @@ derive_vars_merged <- function(dataset,
       select(!!!by_vars_right, !!!replace_values_by_names(new_vars))
   }
 
+  if (!is.null(exist_flag)) {
+    add_data <- mutate(
+      add_data,
+      !!exist_flag := TRUE
+    )
+  }
+
   if (!is.null(missing_values)) {
-    match_flag_var <- get_new_tmp_var(add_data, prefix = "tmp_match_flag")
+    missing_values_var <- get_new_tmp_var(add_data, prefix = "tmp_missing_flag")
   } else {
-    match_flag_var <- match_flag
+    missing_values_var <- missing_values
```

Comment:
I would use
```
missing_values_var <- NULL
```
Technically it makes no difference but I think it is better to read.